naming/src/main/java/com/alibaba/nacos/naming/consistency/ephemeral/distro/DataSyncer.java

```diff
@@ -50,12 +52,14 @@
     private final Serializer serializer;
     private final DistroMapper distroMapper;
     private final ServerMemberManager memberManager;
-
+    private final int firstRetryCount = 1;
+    private final int maxRetryCount = 10;
+    private AtomicInteger requestCount = new AtomicInteger(0);
```

Comment:
two problem:
1. requestCount will be shared between different task. Task A failed, Task B failed, the request count will be 2, but one of them should only wait 1 multiple delay.
2. no reset to 0 for requestCount.